schema/PACK.xsd

```diff
@@ -17,13 +17,16 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 
-  $Date:        1. Dec 2020
-  $Revision:    1.7.2
+  $Date:        31. Jan 2022
```

Comment:
We need to update the copyright year 2013-2022

Reply:
Good catch. Thanks. Fixed.